config/core/test-configs.yaml

```diff
@@ -2130,6 +2137,19 @@ test_configs:
       - baseline
       - baseline-nfs
 
+  - device_type: juno-r0-acpi
+    test_plans:
+      - baseline
+      - baseline-nfs
+      - kselftest-futex
+      - kselftest-lib
+      - kselftest-lkdtm
+      - kselftest-rtc
```

Comment:
Some failures: https://staging.kernelci.org/test/plan/id/623d1df30aeea737045208ff/

Reply:
AFAICT that's a legitimate result, the test is correctly identifying a problem with alarm support in the EFI RTC driver.